sql/ll_bitemporal_list_of_fields.sql

```diff
@@ -31,18 +31,18 @@ CREATE OR REPLACE FUNCTION bitemporal_internal.ll_bitemporal_list_of_fields(p_ta
 AS
 $BODY$
 BEGIN
-RETURN ( array(SELECT attname
+RETURN ( array (SELECT attname
                           FROM (SELECT * FROM pg_attribute
                                   WHERE attrelid=p_table::regclass AND attnum >0) pa
                           LEFT OUTER JOIN pg_attrdef pad ON adrelid=p_table::regclass
                                                         AND adrelid=attrelid
                                                         AND pa.attnum=pad.adnum
-                          WHERE (adbin IS NULL)
-                                AND attname !='asserted'
+                          WHERE ((pg_get_expr(adbin, adrelid) NOT LIKE 'nextval%' and attname not like '%_key') OR adbin is NULL) AND attname !='asserted'
```

Comment:
Let's do it the more proper way than using LIKE. Fetch the table name from relation id and construct the BT sequence column.
```
SELECT relname
FROM pg_catalog.pg_statio_user_tables
WHERE relid = <adrelid>;
```